contracts/upkeeps/CronUpkeep.sol

```diff
@@ -189,6 +186,10 @@ contract CronUpkeep is
       uint256 nextTick
     )
   {
+    if (s_targets[id] == address(0)) {
+      revert CronJobIDNotFound(id);
+    }
```

Comment:
can we refactor this check and the check in `deleteCronJob()` to use a modifier?